src/Sentry/HeapDumpTriggers.cs

```diff
@@ -1,11 +1,16 @@
+using Sentry.Infrastructure;
+
 namespace Sentry;
 
+#if MEMORY_DUMP_SUPPORTED
```

Comment:
to be sure, we set this when we build right?

Reply:
Yeah that gets set here:
https://github.com/getsentry/sentry-dotnet/blob/ed0fd8a1c4383c70797b3b543b23e4f2aa692405/Directory.Build.props#L90-L97